src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Add-AzureRmDataLakeStoreVirtualNetworkRule.md

```diff
@@ -0,0 +1,155 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeStore.dll-Help.xml
+Module Name: AzureRM.DataLakeStore
+online version:
+schema: 2.0.0
+---
+
+# Add-AzureRmDataLakeStoreVirtualNetworkRule
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@lewu-msft please fill in the template sections provided (synopsis, description, examples)